src/transformState.js

```diff
@@ -62,7 +62,21 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
```

Comment:
```suggestion
  for (const { operation, properties } of transforms) {
```
it would be nice to use destructuring here